hazelcast/src/main/java/com/hazelcast/config/Config.java

```diff
@@ -2649,4 +2653,42 @@ public String toString() {
                 + ", metricsConfig=" + metricsConfig
                 + '}';
     }
+
+    /**
+     * Populate Hazelcast <code>Config</code> object from an external configuration file.
+     * It tries to load Hazelcast configuration from a list of well-known locations.
+     * When no location contains Hazelcast configuration then it returns default.
+     *
+     * @return Config created from a file when exists, otherwise default.
+     */
+    public static Config load() {
```

Comment:
not a great name

Reply:
I have no problem with the name. It explains that the config is loaded and this implies some files being checked. Unless you can come up with a better name, I would leave it as it is.